tests/unit/bokeh/core/test_properties.py

```diff
@@ -313,7 +298,7 @@ class NotSerialized(HasProps):
     def test_readonly(self) -> None:
         class Readonly(HasProps):
             x = Int(12, readonly=True)    # with default
-            y = Int(readonly=True)        # without default
+            y = Nullable(Int(), readonly=True)        # without default
```

Comment:
just noting, if this is changed to
```
Nullable(Int(readonly=True))
```
the the property seems to get an implicit default value of zero. That seems perhaps unintended?